Solutions/Endjin.SpecFlow.Selenium.Sample.Specs/WebsiteUnderTest/ReadMe.txt

```diff
@@ -32,6 +32,14 @@ this.AddPage<RegisterPageModel>(Page.Register, "Account/Register");
 <!-- Test website -->
 "website_url": "http://myteswebsite.com/"
 
+<!-- Whether or not the application under test should use a navigation map of page names to page URLs. 
+If true, the package will set the current page model using the current URL and WebsiteNavigationMap on navigation and element click events. 
+If false, the current page model must be set explicitly in your step definitions 
+Setting this option to false allows the library to support single page applications which have a single invariant URL for all views 
+Single page applications which display other route marks (e.g. '#' or '#!') in the URL can use the navigation map
+Single page applications which display other route marks (e.g. '#' or '#!') in the URL should specify the SpaRoutingMarker in the WebsiteNavigationMap. The routing marker should not be included in MapPages() -->
+"use_navigation_map": false
```

Comment:
Shouldn't this be shown as true to match the default setting in the json config (otherwise a bit confusing)